internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go

```diff
@@ -230,6 +232,11 @@ func resourceDataFactoryIntegrationRuntimeAzureSsis() *pluginsdk.Resource {
 								"HS_Gen5_2", "HS_Gen5_4", "HS_Gen5_6", "HS_Gen5_8", "HS_Gen5_10", "HS_Gen5_12", "HS_Gen5_14", "HS_Gen5_16", "HS_Gen5_18", "HS_Gen5_20", "HS_Gen5_24", "HS_Gen5_32", "HS_Gen5_40", "HS_Gen5_80",
 							}, false),
 						},
+						"elastic_pool_name": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: sqlValidate.ValidateMsSqlElasticPoolName,
+						},
```

Comment:
Can we add an conflict with for the `elastic_pool_name` and `pricing_tier` as they both correspond to the same API property?